gateway/probes/views.py

```diff
@@ -4,15 +4,7 @@
 
 
 def readiness(request):
-    try:
-        resp = requests.get(os.environ.get("RAY_HOST", "http://ray") + "/api/jobs/")
-    except Exception as err:
-        print(err)
-        return HttpResponse(status=503)
-    if 200 <= resp.status_code and resp.status_code < 500:
-        return HttpResponse("Hi! I'm ready.")
-    else:
-        return HttpResponse(status=503)
+    return HttpResponse("Hi! I'm ready.")
```

Comment:
Not urgent so I can open an issue for this, but definitely something that we will need to have to pass the security checks. In this end-point the team will ask us to check the status of the database too. So this check is not enough.